packages/expo-localization/src/Localization.ts

```diff
@@ -0,0 +1,23 @@
+import ExpoLocalization from './ExpoLocalization';
+
+import { Localization } from './Localization.types';
+
+export let locale = ExpoLocalization.locale;
+export let locales = ExpoLocalization.locales;
+export let timezone = ExpoLocalization.timezone;
+export let isoCurrencyCodes = ExpoLocalization.isoCurrencyCodes;
+export let country = ExpoLocalization.country;
+export let isRTL = ExpoLocalization.isRTL;
+
+export async function getLocalizationAsync(): Promise<Localization> {
+  const localization = await ExpoLocalization.getLocalizationAsync();
+  locale = ExpoLocalization.locale;
```

Comment:
Accessing ExpoLocalization will cause us to recompute everything and also have inconsistent data -> reading from `localization` looks more correct:

```js
({ locale, locales, timezone, isoCurrencyCodes, country, isRTL } = localization);
```